### PR TITLE
Send access token in headers and support Python 3

### PR DESCRIPTION
**Pull request by bxsx**
#### Major changes:
- send access token in headers instead of query strings
- support Python 3

#### Minor changes:
- refactoring and formatting

## Type of change

- [ ] Bug fix (non-breaking change that fixes an issue)
- [x] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] Maintenance
- [ ] New release

## Related issues

- Fix [SC-92935]

### Development

- [x] Lint rules pass locally
- [ ] The code changed/added as part of this pull request has been covered with tests
- [x] All tests related to the changed code pass in development

### Code review

- [ ] This pull request has a descriptive title and information useful to a reviewer. There may be a screenshot or screencast attached
- [ ] "Ready for review" label attached to the PR and reviewers assigned
- [ ] Issue from task tracker has a link to this pull request
- [ ] Changes have been reviewed by at least one other engineer

### CODE REVIEW

**Comment by bxsx** [comment]
[SC92935]

File: fetch_people_for_items.py [hunk 1]
```diff
@@ -15,62 +15,68 @@
...
         page_number += 1
 
 
 def fetch_page(access_token, item_id, page=1):
-    resp = requests.get('https://api.rollbar.com/api/1/item/%d/instances/?access_token=%s&page=%d' %
-        (item_id, access_token, page))
-    
+    resp = requests.get(
+        "https://api.rollbar.com/api/1/item/{}/instances/?page={}".format(item_id, page),
+        headers={"X-Rollbar-Access-Token": access_token},
```
Comment (bishopb) [inline fetch_people_for_items.py:60]: Nit: This is a faithful refactor, but I notice that the prior `requests.get` denied redirects, but this one does not. I see no reason they should be different. Should this one also set `allow_redirects=False`?

**Review by bishopb — APPROVED** [review]
🔒 security FTW.

File: fetch_people_for_items.py (hunk 1 above)
Comment (bxsx) [reply fetch_people_for_items.py:60]: `301` is the expected status code in the previous response:
https://github.com/rollbar/api-people-example/blob/7ddf2eba23f3eedcd22bf0d0a218b75abecde9f2/fetch_people_for_items.py#L28-L30

This is in line with https://explorer.docs.rollbar.com/#operation/get-an-item-by-project-counter

For [list all occurrences in an item](https://explorer.docs.rollbar.com/#tag/Occurrence/paths/~1api~11~1item~1{item_id}~1instances/get) endpoint, we expect `200` status code in the response:

https://github.com/rollbar/api-people-example/blob/7ddf2eba23f3eedcd22bf0d0a218b75abecde9f2/fetch_people_for_items.py#L57-L59

**Pull request merged**